vm-setup/roles/packages_installation/tasks/main.yml

```diff
@@ -4,32 +4,33 @@
       package:
         name: "{{ packages.ubuntu.common.packages}}"
         state: present
-      become: true
     - name: Install packages using standard package manager for Ubuntu 18.04
       when: ansible_distribution_version == "18.04"
       package:
         name: "{{ packages.ubuntu.bionic.packages}}"
         state: present
-      become: true
     - name: Install packages using standard package manager for Ubuntu 20.04
       when: ansible_distribution_version == "20.04"
       package:
         name: "{{ packages.ubuntu.focal.packages}}"
         state: present
-      become: true
     - name: Install packages using pip3
       pip:
         executable: "pip3"
         name: "{{ packages.ubuntu.pip3 }}"
         state: present
-      become: true
+  become: true
   when: ansible_facts['distribution'] == "Ubuntu"
 
 
 - name: Install packages on CentOS
   block:
     - name: Install packages using standard package manager
       block:
+        - name: Install common packages using standard package manager for Centos
+          package:
+            name: "{{ packages.centos.common.packages }}"
+            state: present
```

Comment:
If we do this it would be better to join the lists as it will be faster than running two install tasks